modAionImpl/src/org/aion/zero/impl/blockchain/AionImpl.java

```diff
@@ -149,6 +149,10 @@ public long estimateTxNrg(AionTransaction tx, IAionBlock block) {
                             LOG_VM,
                             getPostExecutionWork());
             return executor.execute().get(0).getReceipt().getEnergyUsed();
+        } catch (VMException e) {
+            LOG_GEN.error(e.toString());
```

Comment:
similarly, needs the message and stack trace

Reply:
updated